ProcessHacker/proctree.c

```diff
@@ -1211,6 +1211,133 @@ static VOID PhpUpdateProcessNodeAppId(
     }
 }
 
+static ULONG PhpGet_gfDPIAwareOffset32(
```

Comment:
Naming scheme doesn't follow project conventions

Reply:
What would you suggest? PhpGetGfDPIAwareOffset32? The variable is called gfDPIAware so I didn't want to uppercase the g.